core/src/main/scala/spire/math/Interval.scala

```diff
@@ -421,6 +423,9 @@ sealed abstract class Interval[A](implicit order: Order[A]) { lhs =>
       case (_, _) => lhs * rhs.reciprocal
     }
 
+  def /(rhs: A)(implicit ev: Field[A]): Interval[A] =
+    lhs * rhs.reciprocal
```

Comment:
Why use this rather than `lhs / rhs`? I'm worried a bit about using 2 lossy operations for, eg, Double, rather than just 1.